src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+  --color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
```

Comment:
I guess if you delete it tests will pass, otherwise, try to set `margin-right: 20px` instead of 10 for both sides to `nav__item`